### ISSUE TITLE
Fedora disk images created using osbuild include a 'BIOS boot' partition

### ISSUE THREAD

**Issue by paulwhalen** [osbuild/osbuild-composer#3044]

**Comment by gicmo** [comment]
Yeah, we do hybrid boot per default for all our x86_64 images. Would you rather prefer to just have it UEFI only?

**Comment by nullr0ute** [comment]
> Yeah, we do hybrid boot per default for all our x86_64 images. Would you rather prefer to just have it UEFI only?

Yes please, we only support UEFI platforms as it's impossible to secure BIOS platforms.

**Comment by achilleas-k** [comment]
I'll include the change in the existing open PR along with other fixes: #3038
